### PR TITLE
feat: BNGP-3614 capture full name or organisation name of landowner

### PR DESCRIPTION
**Pull request by KiranVarma02**

### CODE REVIEW

File: packages/webapp/src/routes/land/client-individual-organisation.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    if (!landownerType) {
+      return h.view(constants.views.CLIENT_INDIVIDUAL_ORGANISATION, {
+        err: [{
+          text: 'Select if your client is an individual or organisation',
+          href: '#landownerType'
+        }]
+      })
+    }
+    request.yar.set(constants.redisKeys.CLIENTS_ORGANISATION_NAME, landownerType)
```
Comment (teddmason) [inline packages/webapp/src/routes/land/client-individual-organisation.js:18]: If this was my request it needs to be `constants.redisKeys.CLIENT_INDIVIDUAL_ORGANISATION`

Comment (teddmason) [reply packages/webapp/src/routes/land/client-individual-organisation.js:18]: CLIENTS_ORGANISATION_NAME is definitely a mistype i'm sure

File: packages/webapp/src/routes/land/clients-name.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import constants from '../../utils/constants.js'
+import {
+  validateFirstLastName
+} from '../../utils/helpers.js'
+import isEmpty from 'lodash/isEmpty.js'
+
+const handlers = {
+  get: async (request, h) => {
+    return h.view(constants.views.CLIENTS_NAME)
```
Comment (teddmason) [inline packages/webapp/src/routes/land/clients-name.js:9]: Can the GET populate the name fields if pre exisiting, just to help for routing from CYA page?

File: packages/webapp/src/routes/land/client-individual-organisation.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import constants from '../../utils/constants.js'
+
+const handlers = {
+  get: async (request, h) => {
+    return h.view(constants.views.CLIENT_INDIVIDUAL_ORGANISATION)
```
Comment (teddmason) [inline packages/webapp/src/routes/land/client-individual-organisation.js:5]: Can the Get pre-populate to assist with CYA routing?

File: packages/webapp/src/routes/land/clients-organisation-name.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import constants from '../../utils/constants.js'
+
+const handlers = {
+  get: async (request, h) => {
+    return h.view(constants.views.CLIENTS_ORGANISATION_NAME)
```
Comment (teddmason) [inline packages/webapp/src/routes/land/clients-organisation-name.js:5]: again pre populate

**Review by teddmason — COMMENTED** [review]
few things to sort pls

File: packages/webapp/src/routes/land/client-individual-organisation.js (hunk 1 above)
Comment (teddmason) [reply packages/webapp/src/routes/land/client-individual-organisation.js:18]: Oh and obviously CLIENT_INDIVIDUAL_ORGANISATION will need adding to the redisKeys in constants too :)

Comment (KiranVarma02) [reply packages/webapp/src/routes/land/client-individual-organisation.js:18]: you are right, its a mistype

File: packages/webapp/src/routes/land/clients-name.js (hunk 2 above)
Comment (KiranVarma02) [reply packages/webapp/src/routes/land/clients-name.js:9]: done

File: packages/webapp/src/routes/land/client-individual-organisation.js (hunk 3 above)
Comment (KiranVarma02) [reply packages/webapp/src/routes/land/client-individual-organisation.js:5]: done

**Review by teddmason — APPROVED** [review]

**Pull request merged**